packages/editor/src/constants.ts

```diff
@@ -46,6 +46,7 @@ export const PATHS = {
   External: '/external-page',
   Heartbeat: '/heartbeat',
   Run: '/run',
+  Auth: '/auth',
```

Comment:
alphabetical that way it's in the same order as the page folders in pages folder